src/app/src/main/kotlin/org/imperial/mrc/hint/emails/RealEmailManager.kt

```diff
@@ -0,0 +1,38 @@
+package org.imperial.mrc.hint.emails
+
+import org.imperial.mrc.hint.AppProperties
+import org.simplejavamail.email.Email
+import org.simplejavamail.mailer.Mailer
+import org.simplejavamail.mailer.config.ServerConfig
+import org.simplejavamail.mailer.config.TransportStrategy
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+class RealEmailManager(appProperties: AppProperties): EmailManager
```

Comment:
I tested that this can actually send emails when configured as for Montagu production, but I guess we need to set up a Hint/Naomi sender, or even a separate email server (depending on where we deploy this!)